### PR TITLE
Fix/5117  header disappear on mobile

### PR DESCRIPTION
**Pull request by Skylsmoi**
https://github.com/tracim/tracim/issues/5117

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Comment by Skylsmoi** [comment]
No tests written since the issue only happens on mobile on which we aren't equipped to create end to end tests yet.

**Review by raphj — APPROVED** [review]

File: frontend/src/container/WorkspaceContent.jsx [hunk 1]
```diff
@@ -185,7 +185,7 @@ export class WorkspaceContent extends React.Component {
 
     if (hasWorkspaceIdChanged || prevFilter !== currentFilter) {
       this.setState({ workspaceIdInUrl: workspaceId, contentLoaded: false })
-      this.loadAllWorkspaceContent(workspaceId, false, hasWorkspaceIdChanged)
+      this.loadAllWorkspaceContent(workspaceId, false)
```
Comment (raphj) [inline frontend/src/container/WorkspaceContent.jsx:188]: whoops, good catch

File: frontend/src/component/FeedItem/FeedItemFooter.jsx [hunk 2]
```diff
@@ -62,7 +62,7 @@ export class FeedItemFooter extends React.Component {
           {props.isPublication && props.showTimeline && (
             <IconButton
               text={props.discussionToggleButtonLabel}
-              textMobile={props.discussionToggleButtonLabelMobile}
+              textMobile={props.discussionToggleButtonLabelMobile.toString()}
```
Comment (Haskaris) [inline frontend/src/component/FeedItem/FeedItemFooter.jsx:65]:
```suggestion
              textMobile={props.discussionToggleButtonLabelMobile}
```
According to your changement on `FeedItemWithPreview`, discussionToggleButtonLabelMobile` is already a string.

**Review by Haskaris — CHANGES_REQUESTED** [review]

File: frontend/src/component/FeedItem/FeedItemFooter.jsx (hunk 2 above)
Comment (Skylsmoi) [reply frontend/src/component/FeedItem/FeedItemFooter.jsx]: done

**Review by Haskaris — APPROVED** [review]

**Comment by inkhey** [comment]
Tested on branch on Xiaomi Mi A1 /  android (/e/ rom) with:
- chrome (rebadged as "Browser")
- fennec (firefox based browser)

**Comment by MariePrandstetter** [comment]
tested on branch

**Pull request merged**